manifests/init.pp

```diff
@@ -8,7 +8,72 @@
 # @example
 #   include puppetwebhook
 #
+# @example Specify parameters
```

Comment:
since there are so many docs, did you check if a proper REFERENCE.md can be generated?

Reply:
I will be happy with a `REFERENCE.md` file generated by `puppet strings` command.
@dhollinger  If i can help about this , ping me on IRC.

Excepted this the PR LGTM

Reply:
REFERENCE. Me should be there now